### PR TITLE
feat(MerchantSDK): Add github actions

### PR DESCRIPTION
**Pull request by motokotomax**
Add github actions for Merchant SDK

### CODE REVIEW

File: .github/workflows/merchant-sdk.check.yml [hunk 1]
```diff
@@ -0,0 +1,78 @@
+name: Check Merchant SDK
+on:
+  push:
+    paths:
+      - 'MerchantSDK/**'
+      - 'HealthAPILibrary/**'
```
Comment (zladzeyka) [inline .github/workflows/merchant-sdk.check.yml:6]:
```suggestion
      - 'HealthAPILibrary/**'
```

File: .github/workflows/merchant-sdk.check.yml [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+    tags-ignore:
+      - '**'
+  pull_request:
+    paths:
+      - 'MerchantSDK/**'
+      - 'HealthAPILibrary/**'
```
Comment (zladzeyka) [inline .github/workflows/merchant-sdk.check.yml:12]:
```suggestion
      - 'HealthAPILibrary/**'
```

File: .github/workflows/merchant-sdk.publish.example.apps.yml
Comment (zladzeyka) [inline .github/workflows/merchant-sdk.publish.example.apps.yml:1]: Let's remove this one for a while, otherwise it will always fail for now

**Review by zladzeyka — CHANGES_REQUESTED** [review]
@motokotomax  thank you. Let's remove a publish example workflow

**Comment by motokotomax** [comment]
> @motokotomax thank you. Let's remove a publish example workflow

thanks for review, all comments fixed

**Pull request merged**